### ISSUE TITLE
bug in subset decomp

### ISSUE THREAD

**Issue by jedwards4b** [NCAR/ParallelIO#1316]
The decomp file https://svn-ccsm-piodecomps.cgd.ucar.edu/trunk/72/piodecomp72tasks02dims01.dat shows an error when using 
the following pioperf.nl on 72 tasks
```
&pioperf
decompfile = '/glade/u/home/jedwards/sandboxes/piodecomps/72/piodecomp72tasks02dims01.dat'
pio_typenames = 'pnetcdf'
rearrangers = 2
nframes=1
nvars = 1
/
```

**Comment by jedwards4b** [comment]
Actually I think that this is a problem in the decomp file - the dof on a given node should be monotonically increasing but is not for example on node 50.   The error in pio is that it didn't flag this as a bad input or reorder field to be monotonic.

**Comment by jedwards4b** [comment]
Fixed by #1318
